### ISSUE TITLE
Manage hex / round tiles

### ISSUE THREAD

**Issue by ghost** [court-jus/yack#4]
Hex and round are classical shape for game components.
Maybe the round one is the easiest to implement, but both of them can be interessting.

For example, increase or decrease size of round token to fit your one sized punch.

**Comment by court-jus** [comment]
Yes or just to extract hex tiles that are not laid out in the good form to be easily cut
